website/docs/developer/development-guide/documentation/documentation-deployment.md

```diff
@@ -4,4 +4,73 @@ title: Deploying Documentation
 desc: Provides basic information on deploying ORCA documentation to nasa.github.io
 ---
 
-Information on deploying the documentation is TBD.
+The following sections provide information to ORCA users to create automated or
+manual releases on github. View the entire versioning and releases document 
+[here](https://github.com/nasa/cumulus-orca/blob/master/docs/release.md).
+
+## Automated Documentation Release on Github
+
+### Create a new ORCA release on github
+
+The release is automated in Bamboo, but the step must be manually started. If 
+you set the `RELEASE_FLAG` to `true` and the build steps passed, you will be 
+able to run the manual "Release" step in Bamboo.
+
+The CI release scripts will create a release based on the release version tag, 
```

Comment:
Remove this block 19-27 as they talk about the code release and not the documentation (website) release. Possibly change to something like:

On a successful automated release the updated documentation is committed to the `gh-pages` branch of the [Cumulus ORCA repo](Address to branch). The updated documentation should be available on the Cumulus ORCA website at https://nasa.github.io/cumulus-orca.

Reply:
Removed 19-27 and replaced with suggested content.